comp/forwarder/defaultforwarder/forwarder_health.go

```diff
@@ -106,6 +106,7 @@ func (fh *forwarderHealth) init() {
 			oldAPIKey, ok1 := oldValue.(string)
 			newAPIKey, ok2 := newValue.(string)
 			if ok1 && ok2 {
+				fh.log.Debug("Updating API key in forwarder, replacing %s with %s", oldAPIKey[:5], newAPIKey[:5])
```

Comment:
Length of the api key is not guaranteed, we should only print the last digit if it's 32 char. We have a function for this in the scrubber If think.

Reply:
HideKeyExceptLastFiveChars: https://github.com/DataDog/datadog-agent/blob/cba18c76a243de712b679664abfdf53a74edb3b3/pkg/util/scrubber/default.go#L334C6-L334C32

Reply:
Oh nice, thanks a lot !